smart-contracts/scripts/getBridgeRegistryAddress.js

```diff
@@ -12,6 +12,8 @@ module.exports = async () => {
     require("../build/contracts/BridgeRegistry.json")
   );
 
+  console.log("Expected usage: \n truffle exec scripts/peggy:address.js --network ropsten");
```

Comment:
This should be scripts/getBridgeRegistryAddress.js instead of what is currently there.

Reply:
wrong script name, will fix it.